.gitignore

```diff
@@ -163,3 +163,7 @@ log/DSP.leaks
 
 #VS Code
 .vscode
+conf/search_server.conf
```

Comment:
um...undo this please

Reply:
any new about this project ?

Reply:
> 
> 
> um...undo this please

Maybe we should rename the config files to "map_darkstar.conf.default" and have server owners copy them to ".conf", that way there's no risk of getting your current configuration overwritten when pulling / having to stash before pushing.